salt/journal-cms/init.sls

```diff
@@ -319,6 +319,14 @@ site-cache-rebuild-again:
         - require:
             - site-configuration-import
 
+site-permissions-rebuild:
+    cmd.run: ../vendor/bin/drush php-eval "node_access_rebuild();"
+        - cwd: /srv/journal-cms/web
+        - user: {{ pillar.elife.deploy_user.username }}
+        - onlyif: cd /srv/journal-cms/web && [[ $(sudo -u {{ pillar.elife.deploy_user.username }} ../vendor/bin/drush php-eval "print node_access_needs_rebuild()") == "1" ]]
+        - require:
+            - site-cache-rebuild-again
```

Comment:
know it's going to be removed soon, but https://github.com/nlisgo/journal-cms-formula/blob/59049a81b812d1e833be54c13c2c92610a377e6a/salt/journal-cms/init.sls#L366 also has
```
require: 
    - site-cache-rebuild-again
```
That one should be changed to be
```
require:
    - site-permissions-rebuild
```
so that this new task is executed between the two. Even after that state (`migrate-content`) is removed, there are further states down the line (e.g. `journal-cms-defaults-users-...`) that will need to be kept after this one.